src/assets/mock-data/match.mock.data.js

```diff
@@ -32,4 +32,37 @@ const mockedMatch = {
   freeTicket: false
 };
 
-module.exports = { mockedMatch };
+const mockedMatchRunning = {
+  id: 1,
+  player1: {
+    id: "PLAYER20",
+    firstname: "Achim",
+    lastname: "Amthor",
+    clubname: "SC Baldham-Vaterstetten",
+    gamesWon: 5,
+    matchIds: [0],
+    qttr: 1351,
+    active: true,
+    hasFreeTicket: false
+  },
+  player2: {
+    id: "PLAYER3",
+    firstname: "Ulrich",
+    lastname: "Dietzel",
+    clubname: "TTC Friedberg",
+    gamesWon: 1,
+    matchIds: [0],
+    qttr: 1111,
+    active: true,
+    hasFreeTicket: false
+  },
+  sets: [
+    { player1: 11, player2: 8 },
+    { player1: 8, player2: 11 },
+    { player1: 10, player2: 12 },
+    { player1: 15, player2: 13 }
+  ],
+  freeTicket: false
```

Comment:
we don't use it anymore

Reply:
so it's only a free ticket player then?

Reply:
yes
Even though there should be never a Freeticket game/player on the client

Reply:
good point!